.circleci/config.yml

```diff
@@ -680,15 +680,15 @@ workflows:
             - api-deps-check
             - api-integration-test
             - ui-unit-test
-      # Not ready
-      # - puppeteer-env-setup
-      # - puppeteer-test:
-          # <<: *filter-release-tags
-          # parallel_num: 4
-          # env_name: "staging"
-          # requires:
-            # - deploy-to-staging
-            # - puppeteer-env-setup
+      - puppeteer-env-setup:
```

Comment:
Unrelated to this PR: I realize I reviewed this last time - but looking at this again I'm not sure about this approach of using a separate job for puppeteer env setup - I think this could just be included as a sub-command within the individual puppeteer jobs instead which would be faster and more resource effective. I'm failing to remember the context at this point, so please let me know if I'm missing something.

Reply:
Sorry, I don't think I explained reasons before. Changes are to make code more efficient.
- To avoid duplicate common setup tasks (steps).
- If a step in `puppeteer-env-setup` fails, it stops the Puppeteer job. Stop early means 4 CI nodes are not used at all. This saves time and money.
- Puppeteer tests in each CI node finish faster without setup tasks.

Reply:
This sounds familiar. Avoiding duplicate work across the 4 parallel tasks is something I was not considering. Sounds fine for now, I'd be curious to see the comparison at some point if you have one though.

Note: this comment is not blocking for this PR, just something I noticed and wanted to inquire about